### ISSUE TITLE
Feature request: JSON converter for output

### ISSUE THREAD

**Issue by moaz-mokhtar** [tijlleenders/ZinZen-scheduler#310]
**Is your feature request related to a problem? Please describe.**
When creating json test cases `input.json` and `output.json`, when got a new issue with `input.json` which need to know the correct output `output.json`, when there is a long start and deadline for list of goals, etc it is complicated and sometimes misleading.

**Describe the solution you'd like**
I can propose a solution which is a converter for `input.json` and `output.json` files into more readable input and output for any one participating in this. So that will unify the way to deal with test cases, and will minimize efforts to create or validate test cases
Another suggestion, to use the frontend part which dealing with `input.json` and `output.json` so it will be used to generate those files from us, so we will not type all contents of `input.json` and `output.json`.

This is not a priority at the moment, but a solution for a problem we will face when there is some rush.

**Comment by tijlleenders** [comment]
@moaz-mokhtar I think this is a great idea!  
I removed the `low prio` label as this is very useful for integrating and tuning the scheduler.  

**For the input generation:**  
We can indeed use the frontend code instead of duplicating the same logic somewhere else.  
We just need to add a frontend function to print the input json to the console. Added #1211 for this.

**For the output validation:**  
Similar to #313 we could include a summary of statistics per goal:  
- Scheduled hours
  - Total
  - Per day
  - Per week
- Time of the day
  - min
  - max
- Days of the week
